src/annotation/annotations/groupDescription.js

```diff
@@ -0,0 +1,5 @@
+export default function groupDescriptions () {
+  return {
+    name: 'groupDescriptions',
+  }
+}
```

Comment:
Could you please just add a comment in here explaining why it was added, and that it's not a real annotation etc.  basically we shouldn't add it, but I don't want to delay your feature further, our tests will need a good refactor anyway.